.github/workflows/test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           cargo test
 
   test-macos:
-    runs-on: macos-latest
+    runs-on: macos-15-intel
```

Comment:
I see in [Standard GitHub-hosted runners](https://docs.github.com/en/actions/reference/runners/github-hosted-runners#standard-github-hosted-runners-for-public-repositories) has `macos-15` which is arm, why choose the smaller subset of the userbase to test for with intel? Would testing both be prudent?

Reply:
`libquil` (the Lisp library) doesn't support arm. So to test macos support we need to use Intel Macs.

Reply:
oh yeah. sad times 😞